core/domain/skill_domain.py

```diff
@@ -212,6 +212,12 @@ def validate(self):
             raise utils.ValidationError(
                 'Expected misconception name to be a string, received %s' %
                 self.name)
+
+        if len(self.name) > 60:
```

Comment:
@seanlip, Can we make skill.name < 50? That way we can use the existing[ utils.require_valid_name](https://github.com/oppia/oppia/blob/a1f3657492a808c50ce16306dd5f6ee1c939c7ad/utils.py#L545) method which handles name validation in other objects as well. 

(Asking you as you have suggested to have it < 60 [here](https://github.com/oppia/oppia/issues/6481#issuecomment-508200257).)

Reply:
Yeah we can do that and we can have the function accept default parameters like if no value for ``name_length`` is passed then the default value i.e 50 can be used.

Reply:
I'm OK with 50 for now, yeah. We can parameterize that function later once we've gotten some experience creating these objects and have a good sense for what the lengths should be.

Reply:
@DubeySandeep Done! but do i have to write several test cases covering all the cases of utils.require_valid_name ?